aeron-test-support/src/main/java/io/aeron/test/cluster/TestCluster.java

```diff
@@ -644,7 +645,14 @@ public AeronCluster connectClient(final CredentialsSupplier credentialsSupplier)
         return connectClient(clientCtx().credentialsSupplier(credentialsSupplier));
     }
 
-    public AeronCluster connectClient(final AeronCluster.Context clientCtx)
+    public AeronCluster connectClient(final AeronCluster.Context clientContext)
+    {
+        return connectClient(clientContext, new RedirectingNameResolver(nodeNameMappings()),
+            true, true);
+    }
+
+    public AeronCluster connectClient(final AeronCluster.Context clientCtx, final NameResolver nameResolver,
+        final boolean keepTryingAfterError, final boolean setIngressEndpoints)
```

Comment:
Please, don't change this API. Instead create necessary objects from scratch in the test.